### DISCUSSION TITLE
User error, install error, or bug?

### DISCUSSION THREAD

**Discussion by bobossler** [leftwm/leftwm#945] [Q&A] [answered]
I get an error when running leftwm-check:

~/.config/leftwm ❯ leftwm-check               
thread 'main' panicked at 'ArgAction::Count is defaulted', /home/bobossler/.cargo/registry/src/github.com-1ecc6299db9ec823/clap-4.0.18/src/parser/matches/arg_matches.rs:144:14
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

~/.config/leftwm ❯ leftwm-check config.ron    
thread 'main' panicked at 'ArgAction::Count is defaulted', /home/bobossler/.cargo/registry/src/github.com-1ecc6299db9ec823/clap-4.0.18/src/parser/matches/arg_matches.rs:144:14
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

~/.config/leftwm ❯ leftwm-check -v config.ron 
thread 'main' panicked at 'ArgAction::Count is defaulted', /home/bobossler/.cargo/registry/src/github.com-1ecc6299db9ec823/clap-4.0.18/src/parser/matches/arg_matches.rs:144:14
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

~/.config/leftwm ❯ leftwm-check -V           
LeftWM Check 0.4.0

~/.config/leftwm ➜ ls             
drwxrwxr-x    - bobossler 29 Oct 02:10 themes
.rw-rw-r-- 5.4k bobossler  1 Nov 23:50 config.ron
.rw-rw-r-- 6.0k bobossler 29 Oct 02:10 themes.toml

I have tried wrapping config.ron in ticks, double quotes, brackets, and even specifying the entire qualified path. I have a basic understanding of rust, but that's it. It appears to me that there is a crate named clap, that is throwing an error while parsing args, but i don't see a problem with the arguments I supplied.

I'm trying to learn and use leftwm, but running into multiple issues. I wanted to use leftwm-check to validate my configuration. I have other theme errors I am working on separately.

Any guidance on my leftwm-check error would be appreciated.

UPDATE: I am also having issues with leftwm-state parsing arguments. I am using the basic_polybar theme and this line in the up script fails:

leftwm-state -q -n -t "$SCRIPTPATH"/sizes.liquid | sed -r '/^\s*$/d' | while read -r width x y

The error returned is:

error: The argument '--template <FILE>' requires a value but none was supplied

The -t option clearly has the sizes.liquid file as an argument.

What can be wrong with my setup for parsing arguments?

**Comment by VuiMuich** [comment, marked as answer]
On which hash is your build based? We had an issue related to clap fixed last week.
Chances are a rebuild from the latest commit might fix your issues.

**Reply by bobossler** [reply]
Thanks! I did a new pull and then a new build and the error is gone.
